### ISSUE TITLE
Windows 10 NVDA Sometimes Reverts To No Audio Ducking

### ISSUE THREAD

**Issue by Elshara** [nvaccess/nvda#5971]
Simply put, if NVDA is set to save audio ducking modes, sometimes it doesn't. Especially when new updates get installed, this functionality gets reversed somehow.

**Comment by jcsteh** [comment]
We're going to need more specific information about how to reproduce 
this. I just tried changing this setting and restarting with several 
different copies of NVDA and cannot reproduce this. Note that if you 
have any configuration profiles, it's possible you have it set 
differently in different profiles and that is causing confusion. Without 
additional information, there is nothing we can do.

**Comment by Elshara** [comment]
I understand. I'll just keep downloading the latest snap shots. Could
be a temporary error on my side.

**Comment by bhavyashah** [comment]
Hi,
I am not any developer, but maybe the log might have useful
information that could assist in tracking down this bug. Do keep an
eye and save NVDA's log when you are able to reproduce this issue the
next time, and share it on this ticket, perhaps that would help in
narrowing down the bug a little...
Thanks.

On 5/21/16, Elshara Silverheart <notifications@github.com> wrote:
> I understand. I'll just keep downloading the latest snap shots. Could
> be a temporary error on my side.
>
> On 20/05/2016, James Teh <notifications@github.com> wrote:
>> We're going to need more specific information about how to reproduce
>> this. I just tried changing this setting and restarting with several
>> different copies of NVDA and cannot reproduce this. Note that if you
>> have any configuration profiles, it's possible you have it set
>> differently in different profiles and that is causing confusion. Without
>> additional information, there is nothing we can do.
>>
>>
>> ---
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub:
>> https://github.com/nvaccess/nvda/issues/5971#issuecomment-220575598
>
>
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/5971#issuecomment-220763992


-- 
Warm Regards
Bhavya Shah
Using NVDA (Non Visual Desktop Access) free and open source screen
reader for Microsoft Windows
To download a copy of the free screen reader NVDA, please visit
http://www.nvaccess.org/
Using Google Talkback on Motorolla G second generation Lollipop 5.0.2
Reach me through the following means:
Mobile: +91 7506221750
E-mail id: bhavya.shah125@gmail.com
Skype id : bhavya.09

**Comment by Elshara** [comment]
Sure I will.

**Comment by ehollig** [comment]
I am not able to replicate this. I set NVDA to Always duck on both a portable and installed copy. Restarting NVDA several times still had the setting for Always duck set. Could we possibly have more information on steps to reproduce the issue or a log set to debug?

**Comment by ehollig** [comment]
Closing as worksforme. If this is still an issue, we can reopen this issue. It may be worth trying to "Reset configuration to factory defaults" to see if something accidentally got changed in your user configuration directory.
